docs/static-analysis.md

```diff
@@ -43,13 +43,17 @@ The plugin for IntelliJ can be found [here](https://plugins.jetbrains.com/idea/p
 > You can [configure all the static analysis tools automatically](#intellij-automatic-setup) or follow the manual instructions.
 
 1. Go to `File → Settings → Other Settings → Checkstyle`.
+1. Set `CheckStyle version` as specified in the build script (`build.gradle`).
 1. Set `Scan Scope` to `Only Java sources (including tests)`.
 1. Click the `+` to add a new configuration file. Click the `Browse` button, navigate to the `static-analysis` folder, and choose the `teammates-checkstyle.xml` file.
 1. Fill in the `Description` field with the name of your project (e.g. teammates).
 1. Click `Next`. Set the value of `basedir` to the path of your project folder.
 1. Click `Finish`.
 1. Check the box next to the newly added rule to activate it.
 
+**NOTE**
+> Once CheckStyle is set-up, the version used will be kept in sync with the build script. See `syncIntelliJCheckStyleVersion` in [Static Analysis Tools Setup Behavior](intellij-automated-setup-behavior.md#static-analysis-tools-setup-behavior).
```

Comment:
It seems like the link is broken :(

Reply:
That's strange. I didn't change the link and tried clicking again and it works, maybe there is a momentary Github bug or something?

Reply:
Ohhh, I thought it should link to the title `versions-sync-with-tools-used-in-build-script` instead of `static-analysis-tools-setup-behavior` and I think it should right :P

Reply:
My bad 😅